### ISSUE TITLE
Feature request: A true dark mode

### ISSUE THREAD

**Issue by mactemporal** [DeaDBeeF-Player/deadbeef#2729]
I just discovered DeaDBeeF, and I'm really impressed and already use It as my primarily music play.
But the light mode is really hurting my eyes.
The only customizable colors are the playlist's colors (AFAIK, sorry if it's not the case).
And also KDE seems to be unable to detect the program for the media keys to work.

OS: Manjaro KDE
DeaDBeeF version: 1.8.8

**Comment by kuba160** [comment]
Deadbeef uses system's gtk theme, ensure that you have chosen dark GTK theme in system settings. For the media keys you need to get MPRIS2 plugin which is not official deadbeef plugin.

**Comment by myself600** [comment]
GTK3 GUI plugin is dark.

**Comment by Kissaki** [comment]
I don't see where GTK system settings would be on Windows. GTK seems to be embedded as a library only in DeaDBeeF.

I was able to make it work. In `DeaDBeeF\share\themes\Windows-10\README.md` I found a reference to the theme source and a dark theme variant ([website](https://b00merang.weebly.com/windows-10.html), [GitHub](https://github.com/B00merang-Project/Windows-10-Dark)).

1. Download dark theme from https://www.opendesktop.org/p/1013482/
2. Extract
3. Rename to `Windows-10-Dark` (drop version number etc)
4. Move `Windows-10-Dark` to `DeaDBeeF\share\themes\` (next to `Windows-10` folder`)
5. Open `DeaDBeeF\etc\gtk-3.0\settings.ini`
6. Change theme name setting `gtk-theme-name = Windows-10-Dark`
7. Restart DeaDBeeF
